src/assets.test.ts

```diff
@@ -31,6 +31,7 @@ describe('assets', () => {
       'mcUSDxOLD.png',
       'mcREAL.png',
       'stabilUSD.png',
+      'portal_ETH.png'
```

Comment:
😭 This icon is very pixelated.

Reply:
super nit: what's up with the `_` in the filename?  it doesn't look like we use that naming convention elsewhere.

Reply:
@silasbw Oh, yeah, sorry, I don't know why I used that name. I'll rename it to `portalETH.png`.
Good catch!